README.md

```diff
@@ -133,9 +133,11 @@ class { '::redis::sentinel':
 }
 ```
 
-### Soft dependency
+### Hard dependency
```

Comment:
Generally we don't document hard dependencies. They're already in metadata.json anyway.